js/app.js

```diff
@@ -0,0 +1,259 @@
+];
+
+//  Use for loops and conditionals and methods in order to complete the below: 
+// Create a new array called bondTitles with only the titles of the Bond films, and console.log the new array.
+
+// create a variable to hold the new array
+let bondTitles = [];
+
+// use a for loop to iterate through the bondFilms array
+for(let i = 0; i < bondFilms.length; i++){
+	// locate the title within the array as we loop through
+	// push those titles to the new array
+	bondTitles.push(bondFilms[i].title)
+}
+// print the new array to the console
+console.log(bondTitles);
+
+// Create a new array oddBonds, of only the Bond films released on odd-numbered years.
+let oddBonds = [];
+// use a for loop to iterate through the bondFilms array
+for(let i = 0; i < bondFilms.length; i++) {
+	// identify all bondFilms that came out in an odd year
+	if (bondFilms[i].year % 2 == 1) {
+		// push those films to the oddBonds array
+		oddBonds.push(bondFilms[i])
+	}
+}
+// print the oddBonds array
+console.log(oddBonds)
+
+// Determine the total cumulative gross of the Bond franchise, and console.log the result. hint To make the grosses into usable numbers, look into the .replace Javascript method (there are many ways to do this, however). Look into parseInt also.
+
+// .replace uses a Regular Expression to match text within a pattern. You can use a metacharacter(which is a character with special meaning) to access a specific character within a pattern (or string in this case). There are lots, but \w can be used to find a word character and \W can be used to find a non-word character. The 'g' refers to a flag - global match, which tells it to find all matches in the string instead of just one (i think). So if there were two $ it would replace both.
+
+// create a variable to put the new value in
+let bondGross = 0;
+// iterate through the array
+for(i = 0; i < bondFilms.length; i++) {
+	// make new variable to hold gross without $ 
+	// replace uses a regExp and what to replace the character/string with as arguments
+	// full note on how .replace works above, also what the heck \/W/ is
+	let newGross = bondFilms[i].gross.replace(/\W/g, '');
+	// turn the gross into a number **NEED RADIX**
+	let numberGross = parseInt(newGross, 10);
+	// add the numbers to get a total
+	bondGross += numberGross;
+}
+
+console.log(bondGross);
+// Dang this franchise has made so much monies: $13,821,621,224.
+
+
+// DIGGING DEEPER
+// declare a variable to hold the object of the actor in the least films
+let leastBond = {}
+// iterate through the object array
+for(let i = 0; i < bondFilms.length; i++) {
+	// find the actor in the array
+	// count how many times each actor appears
+
+}
+
+// compare the number of appearances
+// push the actore with the least appearances to leastBond
+
+// us a for..in loop in a for loop to get the actor? 
+// then a conditional to compare the number of times, the conditional should be nested in a for loop as well
```

Comment:
it's not "nested" unless it's the same thing.  ie a for in a for or an if in an if, etc.